### ISSUE TITLE
DVW32CB Not supported

### ISSUE THREAD

**Issue by seanvictory** [mzdrale/pyubee#14]
Would be great if we could support the DVW32CB model!

**Comment by StevenLooman** [comment]
Hi and thank you for your interest. Adding a model is hard to do without access to it. The least we need is a bit of information which  you can provide by following the [Adding new models guide](https://github.com/mzdrale/pyubee/blob/master/ADDING_NEW_MODELS.md)

But even then, it is fairly hard to do so. It is probably less time consuming for all parties if you add support for it yourself.

**Comment by seanvictory** [comment]
@StevenLooman Done! Please go and review the PR :)

**Comment by StevenLooman** [comment]
Great work! If you fix the comment by @mzdrale it can be merged and a new version can be released.

**Comment by Hukuma1** [comment]
Awesome. Thank you for doing the code!

Any chance we could please also add support for the sister modem, DDW36C?

The specs are pretty much identical. Can readily test if it works as I own this model.

https://www.ubeeinteractive.com/?product=dvw32cb-advanced-wireless-voice-gateway

https://www.ubeeinteractive.com/?product=ddw36c

**Comment by mzdrale** [comment]
@Hukuma1 If you want, you can test. Install `pyubee` and run it:

```
pip install pyubee
```

```
pyubee -m DVW32CB <your_router_IP> <your_router_username> <your_router_password>
```
Example:
```
pyubee -m DVW32CB 192.168.1.1 admin "somepassword"
```

If you get list of connected devices, it means it works. If not, you can try the same command with different model from the list of supported models (currently):  DVW32CB, EVW320B, EVW3226@UPC, EVW32C-0N, EVW321B, EVW3200-Wifi. Let us know if it works for you with any of these models, so we can support your model officially.
If none of them works for you, you can visit [Adding new models document](https://github.com/mzdrale/pyubee/blob/master/ADDING_NEW_MODELS.md). In any case open new issue for supporting DDW36C, as this one is resolved and we are going to close it.

**Comment by StevenLooman** [comment]
Please be aware that adding a new device without access to the device is very hard. Your best bet is to try to add support yourself.

**Comment by StevenLooman** [comment]
Can we close this issue? I think this is resolved by #17.
